site/src/request_handlers/status_page.rs

```diff
@@ -2,14 +2,16 @@ use std::str;
 use std::sync::Arc;
 
 use crate::api::status;
+use crate::api::status::FinishedRun;
 use crate::db::{ArtifactId, Lookup};
 use crate::load::SiteCtxt;
 
-pub async fn handle_status_page(ctxt: Arc<SiteCtxt>) -> status::Response {
-    let idx = ctxt.index.load();
-    let last_commit = idx.commits().last().cloned();
+// How many historical (finished) runs should be returned from the status API.
+const FINISHED_RUN_COUNT: u64 = 5;
```

Comment:
Can history be ~ twice bigger? 5 entries for 2 kind of runs kinda small, plus that way more easy to see any correlation in run time.

Reply:
For future reference, it takes about ~40-50ms to load the status page on my PC/connection in the current version.